content/docs/querying/basics.md

```diff
@@ -107,6 +107,30 @@ a `job` label set to `prometheus`:
 
     http_requests_total{job="prometheus"}[5m]
 
+### Offset modifier
+
+The `offset` modifier allows changing the time offset for individual
+instant and range vectors in a query.
+
+For example, the following expression returns the value of `foo` 5
+minutes in the past relative to the current query evaluation time:
+
+    foo offset 5m
```

Comment:
(actually, s/api_http_requests_total/http_requests_total/, since that is the example metric used multiple times above.